cards/card_component.js

```diff
@@ -26,6 +31,34 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
         })
       );
     }
+
```

Comment:
nit: can we move the registration of these listeners into a function called something like: `addFeedbackListeners()`?